src/libs/actions/EmojiPickerAction.ts

```diff
@@ -0,0 +1,77 @@
+import {ValueOf} from 'type-fest';
+import React from 'react';
+import {View} from 'react-native';
+import CONST from '../../CONST';
+
+type AnchorOrigin = {
+    horizontal: ValueOf<typeof CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL>;
+    vertical: ValueOf<typeof CONST.MODAL.ANCHOR_ORIGIN_VERTICAL>;
+};
+
+type EmojiPicker = {
+    showEmojiPicker: (onModalHideValue?: () => void, onEmojiSelectedValue?: () => void, emojiPopoverAnchor?: View, anchorOrigin?: AnchorOrigin, onWillShow?: () => void, id?: string) => void;
+    isActive: (id: string) => boolean;
+    hideEmojiPicker: (isNavigating: boolean) => void;
+    isEmojiPickerVisible: boolean;
+    resetEmojiPopoverAnchor: () => void;
+};
+
+const emojiPickerRef = React.createRef<EmojiPicker>();
```

Comment:
I just want to mention I have no idea why we keep references to components outside the component tree.

I asked [a question](https://expensify.slack.com/archives/C01GTK53T8Q/p1694031987704399) about this in the #expensify-open-source slack channel.

This is not part of this PR, but we probably should stop doing this pattern.

Reply:
Yes, mee too. It's really odd to me 👍